xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -1184,11 +1184,13 @@ bool CGUIWindowVideoBase::OnPlayAndQueueMedia(const CFileItemPtr& item, const st
   return CGUIMediaWindow::OnPlayAndQueueMedia(movieItem, player);
 }
 
-void CGUIWindowVideoBase::PlayMovie(const CFileItem *item, const std::string &player)
```

Comment:
this is a bad sign. Play should play and keep the item as is (so const). Please check where this is called and set the property there

Reply:
done